### PR TITLE
Add target AOCODARCF7DUAL

### PR DESCRIPTION
**Pull request by star624**
hello,please help to add new target for SJET.
  thanks for your work!

### CODE REVIEW

File: configs/default/SJTE-AOCODARCF722.config [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# Betaflight / AOCODARCF722 (AOF7) 4.3.0 Mar  3 2022 / 17:36:26 (ccd9508fc) MSP API: 1.44
```
Comment (Asizon) [inline configs/default/SJTE-AOCODARCF722.config:1]: Here goes what cpu is going to use.

File: configs/default/SJTE-AOCODARCF722.config [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+# master
+set gyro_to_use = BOTH
+set mag_bustype = I2C
+set mag_i2c_device = 2
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set adc_device = 1
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set dshot_bitbang = AUTO
```
Comment (Asizon) [inline configs/default/SJTE-AOCODARCF722.config:114]: This is a default, no need to define

Comment (Asizon) [inline configs/default/SJTE-AOCODARCF722.config:114]: This is a default, no need to define

File: configs/default/SJTE-AOCODARCF722.config [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+# pin A00: DMA1 Stream 5 Channel 3
+
+# feature
+feature OSD
+
+# serial
+serial 20 1 115200 57600 0 115200
+
+# master
+set gyro_to_use = BOTH
```
Comment (Asizon) [inline configs/default/SJTE-AOCODARCF722.config:106]: Doesnt work with 2 diferent gyros.

Comment (Asizon) [inline configs/default/SJTE-AOCODARCF722.config:106]: Doesnt work with 2 diferent gyros.

**Review by Asizon — CHANGES_REQUESTED** [review]

File: configs/default/SJTE-AOCODARCF722.config (hunk 3 above)
Comment (star624) [reply configs/default/SJTE-AOCODARCF722.config]: i'm very sorry! Step CLI "diff all bare" is missing.
I have changed it,please check again.
thank you.

**Review by Asizon — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**